test-requirements.txt

```diff
@@ -5,7 +5,7 @@ pytest-xdist==1.26.1
 pytest-cov==2.6.1
 fastparquet>=0.1.6
 graphviz>=0.8.4
-moto>=1.3.13
+moto[all]>=1.3.13
```

Comment:
if we're going to set a minimum requirement let's use the latest (1.3.15).  It's the first version to introduce the optional requirements, and since this is the test requirements we can be more aggressive about requiring the latest versions

Reply:
sounds good